src/sourmash/sourmash_args.py

```diff
@@ -391,14 +391,22 @@ def load_pathlist_from_file(filename):
     try:
```

Comment:
note that the try/except should probably be _just_ around the `with open...` block. Generally speaking we want to avoid catching exceptions over-broadly - if we wrap a dozen lines of code in a try/except, we won't know where the error comes from and won't be able to report the problem to the user! (This is a weak rule of thumb, there are some situations where you can't help it, of course. Just pointing it out as an option for you to consider!)

Reply:
Ahh I see... Where can I put the if statements instead?

Reply:
outside the try/except block should work? That may not work for other reasons, but the way scoping works in Python `file_list` will be available outside of the try/except block, if I'm intuiting your concern correctly.